proptest is missing due to the original code using a git dependency for it.

diff --git a/Cargo.toml b/Cargo.toml
index 0cbc976..d415367 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -19,4 +19,7 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/cameron1024/unarray"
 
 [dev-dependencies.test-strategy]
-version = "0.2"
+version = "0.3"
+
+[dev-dependencies.proptest]
+version = "1"
